dwitter/templates/dweet/dweet.html

```diff
@@ -235,6 +234,11 @@
 
       try {
         stats.begin();
+        if ((u + "").match(/c\s*\.\s*width\s*(=|\+=|-=|\*=|\/=|%=|\**=|<<=|>>=|>>>=|&=|\^=|\|=)/) != null) {
+            x.beginPath();
+            x.resetTransform();
+            x.clearRect(0, 0, c.width, c.height);
+        }
```

Comment:
This wouldn't work on the very rare case of `c.height` (I've seen few dweets using it, but can't remember off the top of my head).
Besides, I think something like
```javascript
c.addEventListener("resize", function() { ... });
```
or
```javascript
c.onresize = function() { ... };
```
would be better I think, but I don't have any Windows machine to test this on at the moment.

Reply:
@PythEch  There's no resize event on anything other than the window. Fixing it shouldn't be hard, I only need to change `c\.width` to `(c\.width|c\.height)`

Reply:
You're right, totally forgot about that. Maybe adding a Proxy or defining a getter/setter would be better then. Or we could inspect how jQuery defines resize event on canvas. 

It just feels wrong IMO doing this for every frame for every other browser for performance reasons. 

On the correctness side, it looks like now you can just do:
```javascript
for(...)x.beginPath('c.width='),...
```
or
```javascript
//c.width=
```

now these two examples do not reduce the character count so they are not exploitable, debatable if it's worth the trouble but I could come up with at least one scenario where it reduces char count by 1:
```javascript
x.beginPath(x.fillStyle=`hsl(${i*t},99%,39%/*c.width=`)
```
compared to
```javascript
c.width|=0
x.beginPath(x.fillStyle=`hsl(${i*t},99%,39%`)

you could argue that such an advanced user could already do something like
```javascript
for(c.width|=i=9;i--;)...
```
to save 1 char but not every dweet is applicable for that.